multi_extract.sh

```diff
@@ -102,9 +116,21 @@ done
 #                 Find all rosbags at datadir and extract data                 #
 # ---------------------------------------------------------------------------- #
 echo "\n\nSearching DATA_DIR for ROSBAGS now...\n"
-sleep 5s
-find "$DATA_DIR" -iname "*.db3" -print0 | xargs -0 -I file dirname file | sort | uniq | while read d; do
-    ROSBAG_NAME=$(basename "$d")
+root_dir=$(pwd)
+sleep 2s
+# find "$DATA_DIR" \( -iname "*.db3" -o -iname "*.mcap" \) -print0 | xargs -0 -I file dirname file | while read d; do
```

Comment:
Why leave it commented? What you have added right after seems to be an improvement. If this is for backwards compatibility, either version the script or add a flag? Preference is to delete